splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/GrantRevokeConstantOperation.java

```diff
@@ -55,23 +59,38 @@ public void executeConstantAction( Activation activation ) throws StandardExcept
         LanguageConnectionContext lcc = activation.getLanguageConnectionContext();
         TransactionController tc = lcc.getTransactionExecute();
 
-		List <PermissionsDescriptor> permissionsDescriptors = privileges.executeGrantRevoke( activation, grant, grantees);
+        Manager manager = EngineDriver.driver().manager();
```

Comment:
Try to get ColPermsManager from Manager. If enterprise code is not enabled, an exception will be thrown, and you don't need to add another exception.